Sources/MapboxMaps/Annotations/AnnotationOrchestratorImpl.swift

```diff
@@ -53,14 +53,21 @@ internal final class AnnotationOrchestratorImpl: NSObject, AnnotationOrchestrato
         longPressGestureRecognizer.addTarget(self, action: #selector(handleDrag(_:)))
         longPressGestureRecognizer.delegate = self
         tapGestureRecognizer.delegate = self
+        longPressGestureRecognizer.isEnabled = false
+        tapGestureRecognizer.isEnabled = false
     }
 
     /// Dictionary of annotation managers keyed by their identifiers.
     internal var annotationManagersById: [String: AnnotationManager] {
         annotationManagersByIdInternal
     }
 
-    private var annotationManagersByIdInternal = [String: AnnotationManagerInternal]()
+    internal var annotationManagersByIdInternal = [String: AnnotationManagerInternal]() {
```

Comment:
What was the reason to change the scope from `private` to `internal`?

Reply:
I think I had copied the change from previous version of the file. Will change back